<launch>
  <include file="$(find dogsim)/launch/base.launch" />
  <node pkg="dogsim" type="robot_driver" name="robot_driver" output="screen">
    <param name="solo_mode" value="true"/>
  </node>
</launch>
